clients/mobile/__tests__/__mocks__/reactNavigation.mock.ts

```diff
@@ -1,19 +1,30 @@
 const mockedNavigate = jest.fn()
 const mockDispatch = jest.fn()
+const mockReset = jest.fn()
 
 jest.mock('@react-navigation/native', () => {
+  let routeParams = {}
```

Comment:
I don't see why we need to do it like this, why can't we use returns where we need it?

Reply:
Updated to const [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/202/commits/ba27593e9bda7d902822fc85881238bf5b32500a)